fanuc_lrmate200id_support/urdf/lrmate200id_macro.xacro

```diff
@@ -117,41 +152,47 @@
       <child link="${prefix}link_1" />
       <axis xyz="0 0 1" />
       <limit effort="0" lower="-2.965" upper="2.965" velocity="7.85" />
+      <dynamics damping="1.0" friction="1.0"/>
```

Comment:
Without setting dynamics `1`, the links are decomposed after some motions on Gazebo.